styles/variables-silicon.less

```diff
@@ -232,7 +232,7 @@
 @agate-panel-h-padding: extract(@agate-panels-panel-padding, 2);
 @agate-panels-partial-width: 890px;
 @agate-panels-breadcrumb-width: 105px;
-@agate-panels-controls-top: (extract(@agate-panels-panel-padding, 1) + 41px);
+@agate-panels-controls-top: (extract(@agate-panels-panel-padding, 1) - 17px);
```

Comment:
`(extract(@agate-panels-panel-padding, 1) - 17px)` might not be multiples of 3 because it is 50px - 17px = 43px.

In addition, it's not the scope in this PR, but could you update multiples of 3 in the numbers of `@agate-panels-panel-padding`?
```
@agate-panels-panel-padding: 50px 60px 50px 40px;
```

Reply:
You may have miscalculated 50px - 17px = 43px its actually 33px 😅 
Anyways, I'll change these values to be multiples of 3. Thanks!

Reply:
Yep !! 😅